src/UseCases/AddComment/AddCommentResponse.php

```diff
@@ -7,21 +7,24 @@
 /**
  * @license GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
  */
 class AddCommentResponse {
 
-	public static function newSuccessResponse(): self {
-		return new self();
+	public static function newSuccessResponse( string $successMessage ): self {
+		return new self( '', $successMessage );
 	}
 
 	public static function newFailureResponse( string $errorMessage ): self {
 		return new self( $errorMessage );
 	}
 
 	private $errorMessage;
+	private $successMessage;
 
-	private function __construct( string $errorMessage = '' ) {
+	private function __construct( string $errorMessage = '', string $successMessage = '' ) {
```

Comment:
Now there are 3 empty strings while we only need two. I suggest making the constructor parameters required.